src/style.rs

```diff
@@ -15,14 +15,14 @@ use unicode_segmentation::UnicodeSegmentation;
 pub enum ProgressFinish {
     /// Finishes the progress bar and leaves the current message.
     /// Same behavior as calling [`ProgressBar::finish()`].
-    Default,
+    Finish,
```

Comment:
I think it would be clearer if we call this `AndLeave`, what do you think?

Reply:
Sounds good, it's more consistent with the rest, changed it